pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -216,6 +216,9 @@ def __call__(self, value, system):
                     legend['Geom_Type'] = geom_type
             current['Legend'].append(legend)
 
+            # Clean OtherLegend
+            self._clean_other_legend(current['OtherLegend'], legend)
```

Comment:
For me it's clearer like this ...
```py
# Clean OtherLegend
current['OtherLegend'] = [legend_element for legend_element in current['OtherLegend'] \
    if other_legend['SymbolRef'] != legend_element['SymbolRef']]
```

Reply:
Oh, the otherLegend array is already complete, so you're right. Thanks :-)